### PR TITLE
HTML-CSS-Popup task

### PR DESCRIPTION
**Pull request by Dovahkiin1991**
### HTML-CSS-Popup

[Demo](https://dovahkiin1991.github.io/html-css-popup/) |
[Code base](https://github.com/Dovahkiin1991/html-css-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by Dovahkiin1991** [comment]
Hey, my code meets all requirements.

File: submissions/Dovahkiin1991/html-css-popup/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+.navbar__sub--icons .navbar__link {
+    border: 1px solid transparent;
+}
+
+.navbar__sub--icons .navbar__link:hover,
+.navbar__sub--icons .navbar__link:focus {
+    border-color: #000;
+}
+
+.navbar__item input:checked ~ .navbar__sub {
```
Comment (A-Ostrovnyy) [inline submissions/Dovahkiin1991/html-css-popup/css/styles.css:112]: Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)
Also, avoid too specific CSS selectors. [Read about it](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

File: submissions/Dovahkiin1991/html-css-popup/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    text-align: center;
+}
+
+.navbar__sub--icons .navbar__item--full-width:hover {
+    text-decoration: underline;
+    border-color: transparent;
+}
+
+.navbar__sub--icons .navbar__link {
+    border: 1px solid transparent;
```
Comment (A-Ostrovnyy) [inline submissions/Dovahkiin1991/html-css-popup/css/styles.css:104]: :fire:

File: submissions/Dovahkiin1991/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+            <ul class="navbar">
+                <li class="navbar__item">
+                    <input type="checkbox" id="navbar1">
+                    <label class="navbar__link" for="navbar1">
+                        <img src="./images/icons/popup-button.png" alt="Popup Icon">
+                    </label>
+                    <ul class="navbar navbar__sub navbar__sub--icons">
+                        <li class="navbar__item">
+                            <a href="#" class="navbar__link">
+                                <img src="./images/icons/calendar.png" alt="Calendar Icon">
```
Comment (A-Ostrovnyy) [inline submissions/Dovahkiin1991/html-css-popup/index.html:22]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/Dovahkiin1991/html-css-popup/index.html [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+<body>
+    <header class="header header--sticky">
+        <nav>
+            <ul class="navbar">
+                <li class="navbar__item">
+                    <input type="checkbox" id="navbar1">
+                    <label class="navbar__link" for="navbar1">
+                        <img src="./images/icons/popup-button.png" alt="Popup Icon">
+                    </label>
+                    <ul class="navbar navbar__sub navbar__sub--icons">
```
Comment (A-Ostrovnyy) [inline submissions/Dovahkiin1991/html-css-popup/index.html:19]: Please fix the popup size, it's hard to understand what content is inside the popup
image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

File: submissions/Dovahkiin1991/html-css-popup/index.html (hunk 3 above)
Comment (Dovahkiin1991) [reply submissions/Dovahkiin1991/html-css-popup/index.html:22]: Removed the word "icon" from icons alt text

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**